source/operator.rst

```diff
@@ -163,6 +163,21 @@ The ``upgradeOptions`` section in the `deploy/cr.yaml <https://github.com/percon
 | **Description** | Scheduled time to check for updates, specified in the					|
 |                 | `crontab format <https://en.wikipedia.org/wiki/Cron>`_					|
 +-----------------+---------------------------------------------------------------------------------------------+
+|														|
++-----------------+---------------------------------------------------------------------------------------------+
+|                 | .. _upgradeoptions-setfcv:									|
+|                 |												|
+| **Key**         | `upgradeOptions.setFCV <operator.html#upgradeoptions-setfcv>`_				|
++-----------------+---------------------------------------------------------------------------------------------+
+| **Value**       | boolean											|
++-----------------+---------------------------------------------------------------------------------------------+
+| **Example**     | ``false``											|
++-----------------+---------------------------------------------------------------------------------------------+
+| **Description** | Turns off or on the 									|
+|                 | `FeatureCompatibilityVersion (FCV)								|
+|                 | <https://docs.mongodb.com/manual/reference/command/setFeatureCompatibilityVersion/>`_	|
+|                 | for major version upgrades									|
++-----------------+---------------------------------------------------------------------------------------------+
```

Comment:
I don't like the wording here. It does not turn off/on the FCV. If enabled FCV will be set to match the version during major version upgrade.